.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -63,6 +63,7 @@ schedules:
 variables:
   buildConfiguration: Release
   dotnetCoreSdk5Version: 5.0.401
```

Comment:
I don't think this should be used anywhere any more?

Reply:
Removed in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805